sponsorship-prospectus.md

```diff
@@ -0,0 +1,180 @@
+- Estimated Participation: 1500+
+- Online Community Presence
+  - Local User groups : 10,000+ (via different local mailing lists)
+  - Facebook: 6500+ (likes and followers)
+  - Twitter: 9000+
+
+
+<br>
+
+## Why Should You Sponsor?
+
+### Put Your Business in the Spotlight
+
+Be a part of India’s Premiere conference on using and
+developing the Python Programming Language, increasing your
+brand exposure and elevating your business identity within the
+community. Stand out among the attendees by prominently
+displaying your products and marketing material.
+
+### Hiring and Business Opportunities
+
+Get matched with those who could become potential clients or
+employees. The conference has a history of attracting smart
+folks and businesses from various backgrounds. Get connected,
+make your pitch and find the best programmers or clients to
+join your ranks!
+
+### Give Back to the Community
+
+Establish your goodwill and show the community that you’re a
+reliable partner, willing to support the
+ecosystem. Sponsorship is a way of giving back to the
+community and thanking them for their support. Your
+sponsorship helps PyCon India remain affordable and accessible
+to the widest possible audience.
+
+### Low Cost Marketing
+
+Instead of spending a fortune on traditional advertisements,
+take a look at our reasonably priced sponsorship slabs and
+pick one - letting you be discovered and known to the cream of
+the community at maybe half (or even less!) of your marketing
+budget.
+
+### Flexibility
+
+Have something specific in mind? Please don’t hesitate to ask!
+We are more than happy to work with you and tailor the
+sponsorship benefits to suit your needs.
+
+
+### Audience profile
+
+PyCon India is attended by a diverse mix of people across
+domains and expertise which includes and not limited to -
+Developers, technologists, programmers, educators, bloggers,
+authors, web developers, CTOs, managers, entrepreneurs,
+scientists, engineers and domain experts.
+
+<br>
+
+## Sponsorship Packages
+
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <td colspan="1"><strong>Package</strong></td>
+      <td colspan="2"><strong>Price</strong></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td><strong>INR</strong></td>
+      <td><strong>USD</strong></td>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Diamond</td>
+      <td>₹ 6,00,000</td>
+      <td>$ 8000</td>
+    </tr>
+    <tr>
+      <td>Platinum</td>
+      <td>₹ 4,50,000</td>
+      <td>$ 6000</td>
+    </tr>
+    <tr>
+      <td>Gold</td>
+      <td>₹ 3,00,000</td>
+      <td>$ 4000</td>
+    </tr>
+    <tr>
+      <td>Silver</td>
+      <td>₹ 1,50,000</td>
+      <td>$ 2000</td>
+    </tr>
+    <tr>
+      <td>Startup</td>
+      <td>₹ 50,000</td>
+      <td>$ 700</td>
+    </tr>
+  </tbody>
+</table>
+
+The above mentioned sponsorship amount is exclusive of 18% GST.
+
+<br>
+
+## Sponsorship Benefits
+
+<table class="table table-striped">
```

Comment:
Same here, this table is also rendering without any borders in the current build. Also, the Startup stuff is missing.

Reply:
startup part is fixed now.